source/lac/trilinos_epetra_communication_pattern.cc

```diff
@@ -45,7 +45,7 @@ namespace LinearAlgebra
       comm = std_cxx11::make_shared<const MPI_Comm>(communicator);
 
       Epetra_Map vector_space_vector_map = vector_space_vector_index_set.make_trilinos_map(*comm,
-                                           false);
+                                           true);
```

Comment:
I merely did this because of the test `tests/trilinos/copy_to_dealvec.cc` includes a case where the trilinos vector is ghosted. I agree that it was a bad idea to change this. I will rather disallow this in the test. (Breaking a niche functionality like this straight away without a transition period with deprecated functions seems acceptable, just as we do for the `operator==`, `operator!=`, and `is_non_negative` methods.)